team/src/components/ChannelListContainer/ChannelListContainer.css

```diff
@@ -0,0 +1,75 @@
+.channel-list__container {
+  display: flex;
+  height: 800px;
+  box-shadow: inset 1px 0px 0px rgba(0, 0, 0, 0.1);
+}
+
+.channel-list__sidebar {
+  width: 72px;
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
+    rgb(var(--primary-color));
+  box-shadow: 1px 0px 0px rgba(0, 0, 0, 0.25);
+  border-top-left-radius: 16px;
+  border-bottom-left-radius: 16px;
+}
+
+.channel-list__sidebar__icon1 {
+  width: 44px;
+  height: 44px;
+  margin: 14px;
+  background: linear-gradient(
+      150.64deg,
+      rgba(0, 0, 0, 0.1) 12.73%,
+      rgba(0, 0, 0, 0) 89.32%
+    ),
+    #ffffff;
+  border-radius: 9999px;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.33);
+}
+
+.channel-list__list__wrapper {
+  display: flex;
+  flex-direction: column;
+  background: rgb(var(--primary-color));
```

Comment:
if we can find a way to set this variable in a simpler way, we can probably avoid the querys triggered by the slider

Reply:
Addressed in the other comment, but I don't know of a simpler way so let me know if you have ideas, thanks!